packages/governance/src/contractGovernance/governApi.js

```diff
@@ -32,7 +32,7 @@ const makeApiInvocationPositions = (apiMethodName, methodArgs) => {
  * @param {Instance} governedInstance
  * @param {ERef<{ [methodName: string]: (...args: any) => unknown }>} governedApis
  * @param {Array<string | symbol>} governedNames names of the governed API methods
- * @param {ERef<TimerService>} timer
+ * @param {ERef<import('@agoric/time/src/types').TimerService>} timer
```

Comment:
I think it would be worth the investment to get these on the package export:
```suggestion
 * @param {ERef<import('@agoric/time').TimerService>} timer
```
But I don't think that need be a blocker. It stands out like a sore thumb so it'll get addressed.

Reply:
b7987bf2a8451149bd6442f0716b348ce77bede4 should do it